### ISSUE TITLE
Clarify how to sync vault between two clients

### ISSUE THREAD

**Issue by chrisswanda** [keys-pub/app#56]
I had my work and personal computers sharing the vault for almost a year, so that I could keep keys and secrets in sync between the two.  I received a new work machine yesterday, and would like to set it up to sync with my personal machine again.  For some reason I cannot figure out how I got these two to sync in the past and have yet to figure it out again. 

Two things...

1 - What commands in the CLI, precisely, do I need to do to sync these two machines?  Or in the App, what does an end user do with the vault auth phrase after it is generated?  

2 - Can the docs for the app and/or help context in the CLI (keys vault -h) be updated to be easier to understand for others on how to create a vault auth phrase and what to do afterwards?  After generating the auth phrase, there should be some dialog that says "put auth phrase on the remote client...."  

<img width="527" alt="Screen Shot 2021-05-29 at 04 25 19" src="https://user-images.githubusercontent.com/45573533/120065244-ebfda380-c035-11eb-9354-96a679608908.png">

**Comment by chrisswanda** [comment]
OK, I figured it out now.  For some reason I was just having difficulty this morning.

To get a vault passphrase, on the existing computer type ```keys vault auth```.  It will echo out a BIP39 encoded phrase.  That will be needed on the new computer.

From your new computer that does not have a vault associated with it yet, type ```keys auth vault```.   Then put in your vault phrase from the already existing computer.  

I'll leave the issue open, as for not really clear in the documentation, unless you want a new issue opened.

**Comment by gabriel** [comment]
Yeah I never finished the UX or documentation for that feature. It's not very discoverable or clear how to use. Thanks for posting an issue, maybe others will find it, if they need sync.

I'm not actively working on this at the moment though so...
